numba/cuda/compiler.py

```diff
@@ -1185,6 +1129,18 @@ def inspect_types(self, file=None):
         for _, defn in self.overloads.items():
             defn.inspect_types(file=file)
 
+    def inspect_ptx(self, args, nvvm_options={}):
```

Comment:
This can be removed now this isn't going into 0.55+, because it was already deprecated in 0.54.